test/pdfs/issue20232.pdf.link

```diff
@@ -0,0 +1 @@
+https://github.com/user-attachments/files/22140584/test.pdf
```

Comment:
Since the file is called `test` and doesn't seem to contain much, I wonder if we can just use it as-is instead of as a linked test case? (In-repo PDFs are always a bit easier to deal with.)